rewrite-java/src/main/java/org/openrewrite/java/table/CompileErrors.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2024 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.table;
+
+import lombok.Value;
+import org.openrewrite.Column;
+import org.openrewrite.DataTable;
+import org.openrewrite.Recipe;
+
+public class CompileErrors extends DataTable<CompileErrors.Row> {
```

Comment:
I'd expected data table rows to be inserted from `FindCompileErrors`, similar to [the example here](https://github.com/moderneinc/rewrite-recipe-starter/blob/813f0470feb14001727a9da553ad81832dc6f1f5/src/main/java/com/yourorg/ClassHierarchy.java#L54-L58), but I'm not seeing any reference. Did I overlook something?

Reply:
Pushed the change from @vudayani to fix this